bamboo/npm-tag.js

```diff
@@ -0,0 +1,54 @@
+/* eslint no-console: "off" */
```

Comment:
Direct port from the travis-ci script.   Only update is changing the env variable to GIT_TAG